Riku/Riku/GenerateResources.h

```diff
@@ -3,14 +3,18 @@
 #include "IMove.h"
 #include "GameState.h"
 
-class ChangePlayerResources :
+class GenerateResources :
```

Comment:
Jeśli wydobywamy surowiec ze złoża to nazwa ExtractResources lepiej by pasowała.